Support linux-riscv_64 as a platform identifier

**Is your feature request related to a problem? Please describe.**

When using `JRELEASER_PLATFORM_OVERRIDE=linux-riscv_64` for cross-compiled RISC-V 64-bit Linux binaries, JReleaser 1.22.0 rejects the platform and falls back to `linux-x86_64`:

```
[WARN]  Unsupported platform 'linux-riscv_64', using 'linux-x86_64' instead
```

This causes the assemble step to look for binaries in `target/x86_64-unknown-linux-gnu/release` instead of `target/riscv64gc-unknown-linux-gnu/release`, failing with:

```
[ERROR] JReleaser failed after 0.169 s
Unexpected error when copying files
Input does not exist: target/x86_64-unknown-linux-gnu/release
```

The root cause is that `PlatformUtils.OS_ARCHS` includes `riscv` (32-bit) but not `riscv64`:
https://github.com/jreleaser/jreleaser/blob/main/api/jreleaser-utils/src/main/java/org/jreleaser/util/PlatformUtils.java

Meanwhile, `Detector.java` already correctly maps the `riscv64` architecture:
https://github.com/jreleaser/jreleaser/blob/main/api/jreleaser-utils/src/main/java/org/jreleaser/dependencies/os/Detector.java#L200

**Describe the solution you'd like**

Add `riscv64` to the `OS_ARCHS` list in `PlatformUtils.java` so that `linux-riscv_64` is recognized as a valid platform identifier, similar to how `aarch_64`, `x86_64`, etc. are supported.

**Describe alternatives you've considered**

We are currently working around this by manually creating the ZIP archive for the RISC-V target instead of using `jreleaser assemble`. See: https://github.com/sdkman/sdkman-cli-native/actions/runs/21784738134

**Additional context**

This came up while adding RISC-V 64-bit support to [sdkman-cli-native](https://github.com/sdkman/sdkman-cli-native). The `jreleaser.yml` already has the platform replacement configured (`linux-riscv_64: riscv64gc-unknown-linux-gnu`) and the distribution artifact entry, but the platform override is rejected at validation time.

Related previous issue: #1824 (added `x86_32` support for the same project).

--- a/api/jreleaser-utils/src/main/java/org/jreleaser/dependencies/os/Detector.java
+++ b/api/jreleaser-utils/src/main/java/org/jreleaser/dependencies/os/Detector.java
@@ -252,7 +252,7 @@ public abstract class Detector {
             return "riscv";
         }
         if ("riscv64".equals(value)) {
-            return "riscv64";
+            return "riscv_64";
         }
         if ("e2k".equals(value)) {
             return "e2k";

--- a/api/jreleaser-utils/src/main/java/org/jreleaser/util/PlatformUtils.java
+++ b/api/jreleaser-utils/src/main/java/org/jreleaser/util/PlatformUtils.java
@@ -81,7 +81,8 @@ public final class PlatformUtils {
             "ppcle_64",
             "s390_32",
             "s390_64",
-            "riscv"));
+            "riscv",
+            "riscv_64"));
         OS_ARCHS.sort(Comparator.naturalOrder());
     }
 

